packages/wallets/webpack.config.js

```diff
@@ -195,7 +196,13 @@ module.exports = function (env) {
                 classnames: true,
                 'react-router-dom': true,
                 moment: true,
+                '@deriv-com/analytics': '@deriv-com/analytics',
             },
         ],
+        plugins: [
```

Comment:
do we still need this?

Reply:
Ako while running in localhost and serving wallets package, we get process is not defined error, when trying to read 
process.env.REMOTE_CONFIG_URL, so added this line for running locally wallets.